packages/storage-ui/src/Components/Pages/BucketsPage.tsx

```diff
@@ -261,7 +264,7 @@ const BucketsPage = () => {
             >
               <Trans>Size</Trans>
             </TableHeadCell>
-            <TableHeadCell>{/* Menu */}</TableHeadCell>
+            <TableHeadCell />
```

Comment:
I find it useful to understand that this column title is left empty for the menu, otherwise it's not clear why we'd have such an empty one.
```suggestion
            <TableHeadCell>{/* Menu */}</TableHeadCell>
```